Build: Disable several whole directories

On the chopping block include:
 -examples
 -gdbsx
 -Ocaml tests (they dont build correctly)

diff --git a/tools/Makefile b/tools/Makefile
index 0b721ca..f2821d6 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -7,7 +7,6 @@ SUBDIRS-y += libxc
 SUBDIRS-$(FLASK_ENABLE) += flask
 SUBDIRS-y += xenstore
 SUBDIRS-y += misc
-SUBDIRS-y += examples
 SUBDIRS-y += hotplug
 SUBDIRS-y += xentrace
 SUBDIRS-$(CONFIG_XCUTILS) += xcutils
@@ -30,7 +29,6 @@ endif
 SUBDIRS-y += xenpmd
 SUBDIRS-y += libxl
 SUBDIRS-$(CONFIG_X86) += xenpaging
-SUBDIRS-$(CONFIG_X86) += debugger/gdbsx
 SUBDIRS-$(CONFIG_X86) += debugger/kdd
 SUBDIRS-$(CONFIG_TESTS) += tests
 
